pkg/ocicni/util_unsupported.go

```diff
@@ -0,0 +1,19 @@
+// +build !linux
```

Comment:
What are we going to do for Windows?

Reply:
Add support for it? For now it's a stub to get things compiling. Your experience would be most helpful 🤩

Reply:
@vbatts the grand plan is to remove the need for nsenter entirely and just rely on the CNI plugin and/or cached state.  There's really no reason to touch the network namespace from ocicni.

Reply:
@dcbw that sounds best. Is there a timeline on that?